### DISCUSSION TITLE
anybody could supply sample ceph.conf for Windows

### DISCUSSION THREAD

**Discussion by kevinchi** [dokan-dev/dokany#988] [Q&A] [answered]
I can mount ceph on my linux use command below.
mount -t ceph myceph.com:6789:/testuser  /LOCAL-DIR  -o name=testuser,secret=XXXX
I  need mount ceph on my windows. but I dont know how can I configure the ceph.conf。
anybody help?
appreciate your help

**Comment by Liryna** [comment, marked as answer]
This should be asked on the ceph project you are using. Dokan does not offer to mount ceph filesystem by default.
